docker/gunicorn/web2py.conf.py

```diff
@@ -0,0 +1,44 @@
+# ********************************************************************
+# |docname| - gunicorn configuration for web2py / the Runestone Server
+# ********************************************************************
+# This file configures gunicorn to run the web2py-based (old) Runestone Server.
+#
+# See also the `gunicorn config docs <https://docs.gunicorn.org/en/stable/configure.html#configuration-file>`_.
+#
+# Imports
+# =======
+# These are listed in the order prescribed by `PEP 8`_.
+#
+# Standard library
+# ----------------
+import multiprocessing
+import os
+
+# Third-party imports
+# -------------------
+# None.
+#
+# Local application imports
+# -------------------------
+# None.
+#
+#
+# Configuration
+# =============
+# `chdir <https://docs.gunicorn.org/en/stable/settings.html#chdir>`_: Change directory to specified directory before loading apps. Point this to the location of ``wsgihandler.py``, which must be copied to the web2py root directory.
+chdir = os.environ["WEB2PY_PATH"]
+
+# `wsgi_app <https://docs.gunicorn.org/en/stable/settings.html#wsgi-app>`_: A WSGI application path in pattern ``$(MODULE_NAME):$(VARIABLE_NAME)``.
+wsgi_app = "wsgihandler:application"
+
+# `user <https://docs.gunicorn.org/en/stable/settings.html#user>`_: Switch worker processes to run as this user.
+user = "www-data"
+
+# `group <https://docs.gunicorn.org/en/stable/settings.html#group>`_: Switch worker process to run as this group.
+group = "www-data"
+
+# `workers <https://docs.gunicorn.org/en/stable/settings.html#workers>`_: The number of worker processes for handling requests. Pick this based on CPU count.
+workers = multiprocessing.cpu_count()
+
+# `bind <https://docs.gunicorn.org/en/stable/settings.html#bind>`_: The socket to bind. This must match the socket nginx sends to.
```

Comment:
We should add 

`max_requests = 500` or more not sure but in uwsgi I have it at 500 - this is a good way to prevent memory "leaks" in Python to keep things from eventually crashing.
`max_requests_jitter = 30` I pulled 30 out of my hat. but it ensures that everything doesn't restart in lock step.
`timeout=60` - like the harakiri option for uwsgi this restarts processes that haven't produced any output for 60 seconds.

Reply:
Fantastic, thanks. Will do!

Reply:
I'm putting these in a shared config between web2py and fastapi.

Reply:
Are you sure on 500 max requests? That seems like a very small amount of work before a restart...